draft-ietf-httpbis-unprompted-auth.md

```diff
@@ -234,12 +276,21 @@ Its value is an integer between 0 and 65535 inclusive from the IANA "TLS
 SignatureScheme" registry maintained at
 <[](https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#tls-signaturescheme)>.
 
+## The v Parameter {#parameter-v}
```

Comment:
Why not simply send the nonce on the wire? The server could then check that (1) the nonce is as expected and (2) the signature is valid. I realize that would result in more bytes on the wire, but splitting up the nonce/verification just seems odd to me. The `verification` value is like a proxy for the actual message being signed, so I thought to myself, why not just send the actual message that's signed?

Reply:
See other comment thread